tests/queries/0_stateless/01889_clickhouse_client_config_format.reference

```diff
@@ -12,5 +12,9 @@ yml
 2
 yaml
 2
-ini
-Code: 347. Unknown format of '/config_default.ini' config. (CANNOT_LOAD_CONFIG)
```

Comment:
Also it would be good to check if we still receive exception in appropriate cases

Reply:
I found that a correctly formatted INI file parses without issues in YAML, but triggers exceptions in XML parsing, so i delete the INI test case. 
1. I want to use invalid YAML format to trigger an YAML exception in YAML parsing:
```
; This is a INI file comment
max_threads: 2
```
2. The invalid XML format to trigger an XML exception in XML parsing:
```
<!-- This is a XML file comment -->
<invalid tag><invalid tag>
```